comp/otelcol/ddprofilingextension/impl/extension_test.go

```diff
@@ -181,8 +181,67 @@ func TestOSSExtension(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+type fargateSourceProvider struct{}
+
+func (*fargateSourceProvider) Source(ctx context.Context) (source.Source, error) {
+	return source.Source{
+		Kind:       "task_arn",
+		Identifier: "arn:aws:ecs:us-east-1:123456789012:cluster/default",
+	}, nil
+}
+
+func TestOSSExtensionFargate(t *testing.T) {
```

Comment:
Could you add a test on non fargate host?

Reply:
Added a test for regular host.